style.css

```diff
@@ -0,0 +1,578 @@
+.article-footer-social ul {
+  width: 100%;
+  justify-content: space-between;
+}
+
+.more-articles-grid h3 {
+  grid-column: 1 / -1;
+  align-self: center;
+}
+
+.more-articles-grid-item img {
+  width: 100%;
+}
+
+.more-articles-grid-item h6 {
+  text-align: right;
+  color: var(--secondary-text-color);
+  font-weight: normal;
+}
+
+article p {
+  margin: 10px 0;
+}
+
+.more-articles-grid-item p {
+  margin-top: 10px;
+  color: var(--secondary-text-color);
+}
+
+#expand-btn {
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  width: 100%;
+  border: 1px solid black;
+  z-index: 1001;
+  position: fixed;
+  bottom: 0;
+  padding: 20px;
+  background-color: white;
+}
+
+#expand-btn p {
+  margin: 0 30%;
+  text-align: center;
+  text-transform: capitalize;
+}
+
+#expand-btn span {
+  text-transform: uppercase;
+}
+
+#expand-btn i {
+  margin: 0 30px;
+  font-size: 1.2em;
+}
+
+#inflation-grid-container img {
+  height: 150px;
+}
+
+#inflation-grid-container span {
+  margin-right: 2px;
+  color: var(--primary-text-color);
+}
+
+.article-container {
+  width: 550px;
+  margin: 0 auto;
+}
+
+#subscribe-icon-links a {
+  font-size: 15px;
+  font-weight: bold;
+  color: black;
+}
+
+#bottom-footer-links a {
+  font-size: 0.85em;
+  font-weight: light;
+  color: var(--secondary-text-color);
+}
+
+#editors-pick div a {
+  float: right;
+  width: 50%;
+  position: relative;
+  top: 1.2em;
+  font-size: 0.9em;
+}
+
+#author h3 a {
+  text-decoration: underline #aaa;
+}
+
+#bottom-footer-links a:hover {
+  text-decoration: underline;
+}
+
+#author {
+  margin-top: 30px;
+}
+
+#author h3 {
+  padding: 2% 0;
+}
+
+#author h3 a:hover {
+  text-decoration: none;
+}
+
+article {
+  font-size: 1.5em;
+  font-weight: light;
+  line-height: 2em;
+}
+
+#editors-pick {
+  width: 200px;
+  position: absolute;
+  right: 5vw;
+  z-index: 999;
+  top: 2000px;
+}
+
+#editors-pick h4 {
+  margin: 10px 0;
+  padding: 10px 0;
+  border-bottom: 1px solid var(--borders);
+}
+
+#editors-pick div img {
+  float: left;
+  height: 80px;
+}
+
+#editors-pick div {
+  padding: 10px 0;
+  clear: both;
+}
+
+#inflation {
+  margin: 30px auto;
+}
+
+#inflation > p {
+  color: var(--secondary-text-color);
+  margin: 10px 0;
+}
+
+#inflation-grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 10px;
+}
+
+#inflation hr {
+  width: 10%;
+}
+
+#inflation > h3 {
+  color: var(--secondary-text-color);
+  margin: 10px 0;
+}
+
+.author-links {
+  justify-content: space-between;
+  margin: 20px 0;
+}
+
+.author-social a.solid {
+  background-color: transparent;
+  padding: 0;
+  border: none;
+  font-size: 1.7em;
+}
+
+.author-social a.solid:hover {
+  color: #ddd;
+}
+
+#article-header {
+  text-align: left;
+}
+
+#article-header h1 {
+  font-size: 2.5em;
+  margin: 0 20%;
+  padding: 3% 0;
+  font-style: italic;
+}
+
+#article-header figure {
+  width: 100%;
+}
+
+#article-header figure img {
+  width: 100%;
+}
```

Comment:
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).